starter/script.js

```diff
@@ -1 +1,44 @@
-// put code here!
+
+confirm("Are you ready to play fictional football?")
+var QAnswer1
+var QAnswer = prompt("The crowd roars as the teams take the field. Team captains are sent to the middle of the field. (H)eads or (T)ails?")
+
+while(QAnswer != "H" && QAnswer != "T" && QAnswer != undefined) {
```

Comment:
You're off to a great start! I think the guess who is a very ambitious idea, but i like it! Maybe you could revisit it during project 1?

The purpose of this exercise was to practice conditionals. Be sure to scope your work down to avoid burnout in the future. e.g.

```js
var isRaining = confirm("Is it raining outside?")
if( isRaining ){
  console.log("Bring an umbrella.")
} else {
  console.log("No umbrella needed today")
}
```